nilearn/tests/test_masking.py

```diff
@@ -331,7 +335,7 @@ def test_compute_multi_epi_mask():
     assert_array_equal(mask_ab, mask_ab_.get_data())
 
 
-def test_warning_shape(random_state=42, shape=(3, 5, 7, 11)):
+def test_error_shape(random_state=42, shape=(3, 5, 7, 11)):
```

Comment:
Right I didn't properly expand the diff and only saw the "2D X (should be ok) test", sorry for that